librz/type/format.c

```diff
@@ -1682,7 +1688,7 @@ RZ_API int rz_type_format_struct_size(const RzTypeDB *typedb, const char *f, int
 				free(o);
 				return 0;
 			}
-			int newsize = rz_type_format_struct_size(typedb, format, mode, n + 1);
+			int newsize = format_struct_size(typedb, f, format, mode, depth + 1);
```

Comment:
Rename `f` to something descriptive.

Reply:
I can't because it would require renaming in many places, I try to keep changes in the formatting code to the minimum.